src/main/java/org/embl/mobie/lib/create/ui/ProjectsCreatorUI.java

```diff
@@ -461,18 +461,20 @@ public void addOMEZarrDialog() {
             exclusive = gd.getNextBoolean();
             useFileNameAsImageName = gd.getNextBoolean();
 
-            String filePath = getOMEZarrImagePathDialog();
-
-            if ( filePath != null )
-            {
-                addOMEZarr( filePath, datasetName );
-            }
+            // TODO: a Browse button for local folders would be nice
```

Comment:
I agree! If possible, it would be great to keep the file browser for finding local ome-zarr images. It's much harder to find this path manually to paste in.

Reply:
Yes, but we can do this in the future.